plugins/woocommerce/includes/admin/class-wc-admin-post-types.php

```diff
@@ -125,13 +125,13 @@ public function post_updated_messages( $messages ) {
 		$messages['product'] = array(
 			0  => '', // Unused. Messages start at index 1.
 			/* translators: %s: Product view URL. */
-			1  => sprintf( __( 'Product updated. <a href="%s">View Product</a>', 'woocommerce' ), esc_url( get_permalink( $post->ID ) ) ),
+			1  => sprintf( __( 'Product updated. <a id="view-product-link" href="%s">View Product</a>', 'woocommerce' ), esc_url( get_permalink( $post->ID ) ) ),
```

Comment:
I'm not sure if we have any guidelines around `id` naming, but this seems a bit generic... what is an extension already added an element with this `id`?

Maybe it should be a bit more specific, something like `woocommerce-product-updated-message-view-product-link`? (double check our CSS naming guideslines... I imagine that isn't quite right)

Reply:
One more thing I'll add here is that it might be beneficial to move the class name outside of the localized string.  This has 2 distinct advantages:

1. We prevent breaking of translations between versions if we ever need to update or add to the class.
2. It reduces human error from translators in case they misspell a class name.

`<a href="%1$s" class="%2$s">`

Another option is to move the entire link tag out of the localized string, which works well and seems favorable, but for some reason we rarely seem to do it:

https://github.com/woocommerce/woocommerce/blob/34bf2df5e680f84cbb552c654205bfe0ffd8cdf9/plugins/woocommerce/src/Internal/Admin/Notes/AddFirstProduct.php#L70

Reply:
That's a nice approach!